src/css/dialog.css

```diff
@@ -0,0 +1,42 @@
+.modal-body td {
```

Comment:
This file shouldn't be commited. This is a 3rd time this is happening. Could you please just check what you've commimted?